plugins/system_upgrade.py

```diff
@@ -402,6 +407,11 @@ def configure_upgrade(self):
         self.opts.distro_sync = self.state.distro_sync
         self.cli.demands.allow_erasing = self.state.allow_erasing
         self.base.conf.gpgcheck = self.state.gpgcheck
+        for repo in self.base.repos.values():
+            repo._config.gpgcheck().set(
+                dnf.conf.config.PRIO_RUNTIME, repo.id in self.state.gpgcheck_repos)
```

Comment:
`repo.gpgcheck = repo.id in self.state.gpgcheck_repos` seems more readable.

Reply:
Thanks Marek, this is much better. (For some reason, I thought it's not possible this way, but I guess it was when another priority was required.)